lxd/storage_buckets.go

```diff
@@ -745,7 +750,9 @@ func storagePoolBucketKeysGet(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 
-	memberSpecific := false // Get buckets for all cluster members.
+	// If target is set, get buckets only for this cluster members.
+	targetNode := queryParam(r, "target")
```

Comment:
Hi @MusicDin I just want to spend a bit of time with this to check its the right approach as it stands out to me. I
I'm not saying its not right just yet, but it caught me eye :)

Reply:
i need to dig into it and figure out what i was thinking when i did that. :)

Reply:
Maybe because if `memberSpecific` is always `true` then we cannot return the list of keys for the bucket from other host without specifying that target?

Reply:
which i think should be the correct behaviour.

Reply:
I've had a chance to look at this now and its fine, what I didn't see earlier was that this pattern was already used in this file, except that it was using the correct `targetMember` var name rather than `targetNode` here. Please can you change this to targetMember for consistency with this file. Thanks